src/components/AppHeader.js

```diff
@@ -16,17 +8,17 @@ const AppHeader = () => (
       <Box margin={{ left: 'medium' }} align="center">
         <GrommetIcon size="customSmall" color="brand" />
       </Box>
-      <Anchor margin="xsmall" href="#" label="Grommet Tools" />
+      <Anchor color="black" margin="xsmall" href="#" label="Grommet Tools" />
     </Box>
     <Box background="lightGrey" direction="row" justify="center" round>
-      <TextInput a11yTitle="search bar" placeholder="Search Tools" />
+      <TextInput plain a11yTitle="search bar" placeholder="Search Tools" />
       <Button type="submit">
         <Search />
       </Button>
     </Box>
     <Box margin={{ right: 'large' }} justify="end" direction="row" gap="medium">
-      <Anchor href="#" label="Grommet" />
-      <Anchor href="#" label="Feedback" />
+      <Anchor color="black" href="#" label="Grommet" />
+      <Anchor color="black" href="#" label="Feedback" />
```

Comment:
Let's keep the default color of grommet Anchor, remove `color` from all anchor references and check with Marissa if it still looks fine with her, if it's not let's get on a call.

Reply:
I messaged Marisa, I will let you know what she says